src/main/java/nextstep/fp/Lambda.java

```diff
@@ -26,31 +26,18 @@ public void run() {
         }).start();
     }
 
-    public static int sumAll(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            total += number;
-        }
-        return total;
+    public static int sumAll(List<Integer> numbers, Conditional c) {
+        return numbers.stream()
+                .filter(number -> c.test(number))
+                .reduce(Integer::sum)
```

Comment:
```suggestion
                .reduce(0, Integer::sum)
```
reducing 연산에 초깃값을 지정한다면 Optional이 아닌 Integer로 반환 됩니다 :)